requirements.txt

```diff
@@ -1,3 +1,3 @@
-fonduer>=0.7.0,<0.8.0
```

Comment:
Before we merge we should make this >=0.8.2, <0.9.0

Reply:
Good catch! I updated accordingly.